src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -286,7 +297,7 @@
      *                              Google\Auth library.
      * }
      @if xapiClass.hasMissingDefaultOptions
-         * @throws ValidationException Throws a ValidationException if required arguments are missing
+         * @@throws ValidationException Throws a ValidationException if required arguments are missing
```

Comment:
The only way would be to create a new service that didn't have default service address/scopes. Maybe this is worth it? I don't know if other languages will also benefit.